assets/samplesheet.csv

```diff
@@ -0,0 +1,4 @@
+sample,fastq_1,fastq_2
+ERR2172265,test/input/ERR2172265_1.fastq.gz,test/input/ERR2172265_2.fastq.gz
+ERR2172266,test/input/ERR2172266_1.fastq.gz,test/input/ERR2172266_2.fastq.gz
+ERR2172267,test/input/ERR2172267_1.fastq.gz,test/input/ERR2172267_2.fastq.gz
```

Comment:
This file can be removed, as you provided the samplesheet and tests datasets already in the `test_datasets` repository

Reply:
This is referred to in `usage.md`

```
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 ```

Reply:
Ah perfect, it's true it's nice to provide an example